src/Prometheus/RegistryInterface.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Prometheus;
+
+use Prometheus\Exception\MetricNotFoundException;
+use Prometheus\Exception\MetricsRegistrationException;
+
+interface RegistryInterface
+{
+    /**
+     * @return MetricFamilySamples[]
+     */
+    public function getMetricFamilySamples(): array;
+
+    /**
+     * @param string $namespace e.g. cms
+     * @param string $name e.g. duration_seconds
+     * @param string $help e.g. The duration something took in seconds.
+     * @param array  $labels e.g. ['controller', 'action']
+     *
+     * @return Gauge
+     * @throws MetricsRegistrationException
+     */
+    public function registerGauge($namespace, $name, $help, $labels = []): Gauge;
+
+    /**
+     * @param string $namespace
```

Comment:
In some doc blocks you have a sample ( "e.g. cm/s") but here not.

Reply:
That's because the original class has none, the blocks are just replicated from there.

I usually don't go with adding examples, but if we do want them I would do a separate PR where we align all of those and maybe move them only to the interface dropping them from concrete classes?